ci/Jenkinsfile.combined

```diff
@@ -32,6 +32,9 @@ pipeline {
 
   stages {
     stage('Build') {
+      environment {
+        SENTRY_PRODUCTION = "${utils.isReleaseBuild() ? 'true' : 'false'}"
```

Comment:
Why is this in the combined `Jenkinsfile`? It will not affect sub-jobs spawned by this job.